setup.py

```diff
@@ -12,8 +12,8 @@
     include_dirs  = ['http-parser', '/usr/include/libev'],
     define_macros = [('WANT_SENDFILE', '1'),
                      ('WANT_SIGINT_HANDLING', '1')],
-    extra_compile_args = ['-std=c99', '-fno-strict-aliasing', '-fcommon',
-                          '-fPIC', '-Wall', '-Wextra', '-Wno-unused-parameter',
+    extra_compile_args = ['-fno-strict-aliasing',
+                          '-Wall', '-Wextra', '-Wno-unused-parameter',
                           '-Wno-missing-field-initializers', '-g']
```

Comment:
Why change these parameters?

Reply:
Majority are already default compilation options from setup.py so I removed them. The -std=c99 is the one that breaks on my Ubuntu 17.04, by default setup.py/gcc uses gnu99 I think.